src/main/scala/com/typesafe/sbt/packager/distroless/DistrolessPlugin.scala

```diff
@@ -0,0 +1,262 @@
+    }
+
+    val cmd = if (dockerCmd.value.isEmpty) {
+      makeClasspath(baseDirectory, dependencies) ++
+        debugPort.map(makeJdwpDebugger).toSeq ++
+        defineMainClass(compileMainClass, compileDiscoveredMainClasses, log).toSeq
+    } else {
+      dockerCmd.value
+    }
+
+    (
+      makeFromAs(base, "mainstage") +:
+        makeMaintainer((Docker / maintainer).value).toSeq :+
+        makeWorkdir(baseDirectory)
+    ) ++
+      layerIdsAscending.map { layerId =>
+        makeCopyLayerDirect(layerId, baseDirectory)
+      } ++
+      dockerLabels.value.map(makeLabel) ++
+      dockerEnvVars.value.map(makeEnvVar) ++
+      makeExposePorts(dockerExposedPorts.value, dockerExposedUdpPorts.value, distrolessDebuggerPort.value) ++
+      Seq(makeEntrypoint(dockerEntrypoint.value), makeCmd(cmd))
+  }) ++ mapGenericFilesToDocker
+
+  /**
+    * @param dockerBaseImage
+    * @param name
+    * @return
+    *   FROM command
+    */
+  private final def makeFromAs(dockerBaseImage: String, name: String): CmdLike =
+    Cmd("FROM", dockerBaseImage, "AS", name)
+
+  /**
+    * @param maintainer
+    *   (optional)
+    * @return
+    *   LABEL MAINTAINER if defined
+    */
+  private final def makeMaintainer(maintainer: String): Option[CmdLike] =
+    if (maintainer.isEmpty) None else Some(makeLabel(Tuple2("MAINTAINER", maintainer)))
+
+  /**
+    * @param dockerBaseDirectory
+    *   the installation directory
+    */
+  private final def makeWorkdir(dockerBaseDirectory: String): CmdLike =
+    Cmd("WORKDIR", dockerBaseDirectory)
+
+  /**
+    * @param label
+    * @return
+    *   LABEL command
+    */
+  private final def makeLabel(label: (String, String)): CmdLike = {
+    val (variable, value) = label
+    Cmd("LABEL", variable + "=\"" + value + "\"")
+  }
+
+  /**
+    * @param envVar
+    * @return
+    *   ENV command
+    */
+  private final def makeEnvVar(envVar: (String, String)): CmdLike = {
+    val (variable, value) = envVar
+    Cmd("ENV", variable + "=\"" + value + "\"")
+  }
+
+  /**
+    * @param dockerBaseDirectory
+    *   the installation directory
+    * @return
+    *   COPY command copying all files inside the installation directory
+    */
+  private final def makeCopyLayerDirect(layerId: Option[Int], dockerBaseDirectory: String): CmdLike = {
+
+    /**
+      * This is the file path of the file in the Docker image, and does not depend on the OS where the image is being
+      * built. This means that it needs to be the Unix file separator even when the image is built on e.g. Windows
+      * systems.
+      */
+    val files = dockerBaseDirectory.split(UnixSeparatorChar)(1)
+    val path = layerId.map(i => s"$i/$files").getOrElse(s"$files")
+    Cmd("COPY", s"$path /$files")
+  }
+
+  /**
+    * @param exposedPorts
+    * @return
+    *   if ports are exposed the EXPOSE command
+    */
+  private final def makeExposePorts(
+    exposedPorts: Seq[Int],
+    exposedUdpPorts: Seq[Int],
+    jdwpPort: Option[Int]
```

Comment:
This method has a slight change from the one in `DockerPlugin`. If the JDWP port is specified, then it's added to the exposed list.